tidb-cloud/migrate-from-mysql-using-data-migration.md

```diff
@@ -192,38 +192,91 @@ On the **Create Migration Job** page, configure the source and target connection
     - If you use Public IP or VPC Peering, you need to add the Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any).
     - If you use AWS Private Link, you are prompted to accept the endpoint request. Go to the [AWS VPC console](https://us-west-2.console.aws.amazon.com/vpc/home), and click **Endpoint services** to accept the endpoint request.
 
-## Step 3: Choose the objects to be migrated
+## Step 3: Choose migration job type
 
-1. Choose existing data migration, incremental data migration, or both by choosing the checkboxes.
+In the **Choose the objects to be migrated** step, you can choose existing data migration, incremental data migration, or both.
 
-    > **Tip:**
+### Migrate existing data and incremental data
+
+To migrate data to TiDB Cloud once and for all, choose both **Existing data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
+
+### Migrate only existing data
+
+To migrate only the existing data of the source database to TiDB Cloud, choose **Existing data migration**.
+
+### Migrate only incremental data
+
+To migrate only the incrementable data of the source database to TiDB Cloud, choose **Incremental data migration**. In this case, the migration job does not migrate the existing data of the source database to TiDB Cloud, but only migrates ongoing changes of the source database to TiDB Cloud.
+
+In the **Start Position** area, you can specify the following types of start positions for incremental data migration:
+
+- The time when the incremental migration job starts
+- GTID
+- Binlog file name and position
+
+If you do not specify a start position, the migration job will start from the time when the incremental migration job starts.
+
+- **The time when the incremental migration job starts**
+
+    If you select this option, the migration job will start from the time when the incremental migration job starts to migrate ongoing changes of the source database to TiDB Cloud.
+
+- **Specify GTID**
+
+    Select this option to specify the GTID of the source database, for example, `3E11FA47-71CA-11E1-9E33-C80AA9429562:23`. The migration job will start from the specified GTID to migrate ongoing changes of the source database to TiDB Cloud.
+
+    You can run the following command to check the GTID of the source database:
+
+    ```sql
+    SHOW MASTER STATUS;
+    ```
+
+    > **Note:**
     >
-    > - To migrate data to TiDB Cloud once and for all, choose both **Existing data migration** and **Incremental data migration**, which ensures data consistency between the source and target databases.
-    > - To migrate only the existing data of the source database to TiDB Cloud, only choose the **Existing data migration** checkbox.
+    > - Make sure that the GTID is enabled in the source database.
```

Comment:
we have to tell the user how to  enbale the  GTID for different type of databases, such as Aurora,GCP MyQL and self hosted mysql. You can refer to the doc
 https://pingcap.feishu.cn/docx/PDgAdbFDEouI8pxPyLhcNxcJnNf